package.json

```
@@ -358,8 +358,8 @@
     "@sentry/types": "^8.33.1",
     "@sentry/utils": "^8.33.1",
     "@solana/addresses": "2.0.0-rc.4",
-    "@swc/core": "^1.13.2",
-    "@trezor/connect-web": "~9.4.7",
+    "@swc/core": "1.4.11",
```

```suggestion
    "@swc/core": "^1.13.2",
```

This change will probably require an additional commit after running `yarn && yarn dedupe` to update `yarn.lock`, and also another round of `@metamaskbot update-policies`